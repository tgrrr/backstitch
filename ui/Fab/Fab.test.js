import React from 'react'
import renderer from 'react-test-renderer'
import { Fab } from '.'

it('Fab: default', () => {
    const component = renderer.create(
        <Fab>
            <>
                Fab
            </>
        </Fab>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
