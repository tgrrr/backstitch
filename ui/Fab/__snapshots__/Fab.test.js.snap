// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fab: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Fab "
    data-testid="Fab"
  >
    <button
      className="MuiButtonBase-root MuiFab-root"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiFab-label"
      >
        Fab
      </span>
    </button>
  </div>
</div>
`;
